src/PrestaShopBundle/Controller/Admin/AuthorizationController.php

```diff
@@ -0,0 +1,325 @@
+            'showResetPasswordForm' => false,
+            'isInsecureMode' => $isInsecureMode,
+            'canAccessInsecureMode' => $canAccessInsecureMode,
+            'secureUrl' => $secureModeChecker->secureUrl(
+                $this->generateUrl('_admin_login', [], UrlGeneratorInterface::ABSOLUTE_URL)
+            ),
+            'installDirectoryExists' => $installDirectoryExists,
+            'adminDirectoryRenamed' => $adminDirectoryRenamed,
+            'newAdminDirectoryName' => $newAdminDirectoryName,
+            'newAdminDirectoryUrl' => sprintf(
+                '%s%s%s',
+                $tools->getShopDomainSsl(true),
+                $this->configuration->get('__PS_BASE_URI__'),
+                $newAdminDirectoryName
+            ),
+        ];
+
+        if (!$adminDirectoryRenamed || $installDirectoryExists) {
+            $templateVars['showLoginForm'] = false;
+            $templateVars['showForgotPasswordForm'] = false;
+            $templateVars['showResetPasswordForm'] = false;
+        }
+
+        return $this->render('@PrestaShop/Admin/Login/index.html.twig', $templateVars);
+    }
+
+    /**
+     * Get all authorization error messages.
+     *
+     * @return array
+     */
+    private function getErrorMessages()
+    {
+        $employeeDoesNotExistMessage = $this->trans(
+            'The employee does not exist, or the password provided is incorrect.',
+            'Admin.Login.Notification'
+        );
+
+        $missingInformationMessage = $this->trans(
+            'Some identification information is missing.',
+            'Admin.Login.Notification'
+        );
+
+        return [
+            BadCredentialsException::class => $employeeDoesNotExistMessage,
+            UsernameNotFoundException::class => $employeeDoesNotExistMessage,
+            DomainConstraintException::class => $this->trans(
+                'Invalid email address.',
+                'Admin.Notifications.Error'
+            ),
+            EmployeeNotFoundException::class => $this->trans(
+                'This account does not exist.',
+                'Admin.Login.Notification'
+            ),
+            PasswordResetTooFrequentException::class => $this->trans(
+                'You can reset your password every %interval% minute(s) only. Please try again later.',
+                'Admin.Login.Notification',
+                ['%interval%' => $this->configuration->get('PS_PASSWD_TIME_BACK')]
+            ),
+            FailedToSendEmailException::class => $this->trans(
+                'An error occurred while attempting to reset your password.',
+                'Admin.Login.Notification'
+            ),
+            ResetPasswordInformationMissingException::class => $missingInformationMessage,
+            InvalidEmployeeIdException::class => $missingInformationMessage,
+            InvalidPasswordException::class => $this->trans(
+                'The password is not in a valid format.',
```

Comment:
How about _Invalid password format_ instead? Could we add extra information about the requested format?

Reply:
No extra information, let's keep it basic.